infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -34,6 +34,20 @@ You could set the environment variables for variables to stored terraform backen
 `cp backend.tfvars.template backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
+
+To be able to build the Tables and Housetables service images, you will need to ensure Docker is running on your local machine. This is typically done by opening Docker Desktop. You can check for this by running a command like `docker info` in the terminal.
+
+Sometimes, the images may stop building or uploading to the ACR. This is likely because docker has run out of local memory. To fix this, run `docker system prune --volumes` locally.
+
+### Building OpenHouse project
+
+Before building the images with Terraform, run `./gradlew clean build` from the root directory to configure the JAR files for image builds. This only works on Java8.
```

Comment:
```suggestion
### Build OpenHouse project

Run `./gradlew clean build` from the root directory to build the JAR files that we use in docker image.

### Building Docker image artifacts

<ADD commands to do that>

### Upload Docker image to Azure container registry

<ADD commands to do that>
```

Reply:
These other two sections are automatically done with `terraform apply`. The user does not need to do any additional commands here.